.circleci/config.yml

```diff
@@ -96,8 +96,10 @@ commands:
       - run:
           name: Fail if there are vulnerabilities in packages
           command: |
-            /bin/bash -c 'yarn audit --groups dependencies --level high; [[ $? -ge 8 ]] && exit 1 || exit 0'
-
+            if [ -e ~/.tmp/yarn_deps_have_changed ]; then
```

Comment:
This looks good to me, but it will fail if any dependencies change, right? So if dependency `A` starts failing it will not be possible to add or modify any others until `A` is fixed?

It's still an improvement even if it works that way :)

Reply:
yes, very true